### PR TITLE
Add global migration feature

### PR DESCRIPTION
**Pull request by sebastienB12**

### CODE REVIEW

**Comment by phil1618** [comment]
@aduchate This PR should be closed.

File: src/main/java/org/taktik/icure/entities/FrontEndMigration.java [hunk 1]
```diff
@@ -1,6 +1,7 @@
 package org.taktik.icure.entities;
 
 import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.sun.org.apache.xpath.internal.operations.Bool;
```
Comment (aduchate) [inline src/main/java/org/taktik/icure/entities/FrontEndMigration.java:4]: This is not the import you are looking for

File: src/main/java/org/taktik/icure/entities/embed/TypedValue.java [hunk 2]
```diff
@@ -24,6 +24,7 @@
 import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
 import com.fasterxml.jackson.databind.annotation.JsonSerialize;
 import org.taktik.icure.constants.TypedValuesType;
+import org.taktik.icure.entities.base.Encryptable;
```
Comment (aduchate) [inline src/main/java/org/taktik/icure/entities/embed/TypedValue.java:27]: Looks like you added an import you do not use

**Comment by phil1618** [comment]
@aduchate Cette PR ne doit pas être mergée (https://github.com/taktik/icure-backend/pull/212#issuecomment-638435722) mais je ne peux pas la fermée. 
Nous devons d'abord valider notre gestion des FrontEndMigration. 

La PR https://github.com/taktik/icure-backend/pull/210 est par contre relevante.

**Comment by phil1618** [comment]
> Hi Sébastien, it seems to me that many changes in this PR are not related to those global migrations.
> Could you restrict your PR to migrations?

@aduchate as mentioned in https://github.com/taktik/icure-backend/pull/212#issuecomment-645233234 the current PR is irrelevant and should be closed by you. The relevant PR 210 has been merged already

**Pull request closed without merging**